lib/resque/scheduler/lock/base.rb

```diff
@@ -20,7 +20,7 @@ def acquire!
         end
 
         def value
-          @value ||= [hostname, process_id].join(':')
+          [hostname, process_id].join(':')
```

Comment:
this was because i was forking in test and the pid wasn't changing, in retrospect i should have cleared the instance variable before forking instead of not caching. will fix this before shipping

Reply:
I think it would be fine if you kept it like this, but make `hostname` memoize.

Reply:
done